asf_core_data/analysis/data_loading.ipynb

```diff
@@ -0,0 +1,1979 @@
+    "\n",
+    "It is recommended to download the data and read the data from a local directory instead, especially if needed more than just once."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "b95b9167",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # Commented out to reduce runtime\n",
+    "\n",
+    "# england_epc = epc_data.load_raw_epc_data(\n",
+    "#     data_path=LOCAL_DATA_DIR, subset=\"England\", batch=\"newest\"\n",
+    "# )\n",
+    "# england_epc.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "0cf2936d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # Commented out to reduce runtime\n",
+    "\n",
+    "# %%time\n",
+    "# epc_data.load_raw_epc_data(\n",
+    "#     data_path=\"S3\", subset=\"Wales\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "5845fe29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# #Commented out to reduce runtime\n",
+    "\n",
+    "# %%time\n",
+    "# epc_data.load_raw_epc_data(\n",
+    "#     data_path=\"S3\", subset=\"Scotland\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "id": "11149b25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # Commented out to reduce runtime\n",
+    "\n",
+    "# %%time\n",
+    "# epc_data.load_raw_epc_data(\n",
+    "#     data_path=\"S3\", subset=\"England\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "id": "0d40d697",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # Commented out to reduce runtime\n",
+    "\n",
+    "# %%time\n",
+    "# epc_data.load_raw_epc_data(\n",
+    "#     data_path=\"S3\", subset=\"GB\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "828ebb0e",
+   "metadata": {},
+   "source": [
+    "## Load preprocessed EPC data\n",
+    "\n",
+    "### Loading from local dir"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "bd45c96d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(907997, 5)"
+      ]
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "prep_epc = epc_data.load_preprocessed_epc_data(\n",
+    "    data_path=LOCAL_DATA_DIR,\n",
+    "    usecols=['CURRENT_ENERGY_RATING', 'PROPERTY_TYPE', 'BUILT_FORM', 'CONSTRUCTION_AGE_BAND', 'COUNTRY'],\n",
+    "    version=\"preprocessed_dedupl\", subset='Wales'\n",
+    ")\n",
+    "\n",
+    "prep_epc.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "id": "9301c73e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(prep_epc['COUNTRY'].unique())"
```

Comment:
i'm guessing this was just a check you forgot to delete :)

Reply:
Already gone, but yes, thanks!